packages/wallet/src/redux/__tests__/helpers.ts

```diff
@@ -79,9 +79,35 @@ export const expectThisMessageAndCommitmentSent = (
     expect(item.messagePayload.signedCommitment.commitment).toMatchObject(c);
   });
 };
+
+export const expectThisMessageAndTheseCommitmentsSent = (
+  state: SideEffectState,
+  commitments: PartialCommitments,
+  messageType: string,
+) => {
+  expectSideEffect('messageOutbox', state, item => {
+    expect(item.messagePayload.type).toEqual(messageType);
+    expect(item.messagePayload.signedCommitments).toMatchObject(
+      commitments.map(({ commitment }) => ({ commitment })), // This has the effect of ignoring the signature
```

Comment:
Can't wrap my head around this...

Reply:
`commitments` are signed (partial) commitments, so they have the shape `{ commitment: Commitment, signature: string }`

This helper is only concerned with the commitment part of it, and doesn't assert anything about the signature (in keeping with the existing helper for a single commitment)